docs/deployment.rst

```diff
@@ -173,6 +173,14 @@ changing these based on your specific use case.
      * Default: 67108864
      * Importance: medium
 
+   ``fetch.min.bytes``
+     The minimum number of bytes in message keys and values returned by a single request
+     before the timeout of `consumer.request.timeout.ms` passes.
+
+     * Type: int
+     * Default: the value of `consumer.request.max.bytes`
```

Comment:
Should be -1? We should file a jira to avoid this duplication.

Reply:
True, nice catch.